src/main/java/org/apache/commons/imaging/Imaging.java

```diff
@@ -411,12 +417,14 @@ public static ICC_Profile getICCProfile(final InputStream is, final String filen
      * @param filename
      *            Filename associated with image data (optional).
      * @param params
-     *            Map of optional parameters, defined in ImagingConstants.
+     *            Optional parameters, defined in ImagingParameters.
```

Comment:
This is something we should work on after we have added ImagingParameters. IMHO if the parameter is optional, there should be a method overload that doesn't take the parameter.